pulsar_metrics/metrics/drift.py

```diff
@@ -51,23 +47,25 @@ def evaluate(
         """
 
         try:
+            if self._feature_name is not None:
```

Comment:
Change with Ternary Operator 	
ref_column =reference[self._feature_name]   if self._feature_name is not None else  reference
	self._column = current[self._feature_name]   if self._feature_name is not None else   current

Reply:
Done !